[2025-02-28T04:48:35.059+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-28T04:48:35.084+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: example_bash_operator.runme_2 manual__2025-02-28T04:48:26+00:00 [queued]>
[2025-02-28T04:48:35.096+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: example_bash_operator.runme_2 manual__2025-02-28T04:48:26+00:00 [queued]>
[2025-02-28T04:48:35.097+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-02-28T04:48:35.118+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): runme_2> on 2025-02-28 04:48:26+00:00
[2025-02-28T04:48:35.130+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'example_bash_operator', 'runme_2', 'manual__2025-02-28T04:48:26+00:00', '--job-id', '31', '--raw', '--subdir', '/home/***/.local/lib/python3.12/site-packages/***/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmpd4h4skvj']
[2025-02-28T04:48:35.132+0000] {standard_task_runner.py:105} INFO - Job 31: Subtask runme_2
[2025-02-28T04:48:35.138+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=5860) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-28T04:48:35.139+0000] {standard_task_runner.py:72} INFO - Started process 5926 to run task
[2025-02-28T04:48:35.235+0000] {task_command.py:467} INFO - Running <TaskInstance: example_bash_operator.runme_2 manual__2025-02-28T04:48:26+00:00 [running]> on host 151d4861bf1b
[2025-02-28T04:48:35.382+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='example_bash_operator' AIRFLOW_CTX_TASK_ID='runme_2' AIRFLOW_CTX_EXECUTION_DATE='2025-02-28T04:48:26+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-28T04:48:26+00:00'
[2025-02-28T04:48:35.384+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-02-28T04:48:35.385+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-02-28T04:48:35.385+0000] {logging_mixin.py:190} INFO - Current task name:runme_2 state:running start_date:2025-02-28 04:48:35.085287+00:00
[2025-02-28T04:48:35.386+0000] {logging_mixin.py:190} INFO - Dag name:example_bash_operator and current dag run status:running
[2025-02-28T04:48:35.387+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-28T04:48:35.388+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-28T04:48:35.389+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo "example_bash_operator__runme_2__20250228" && sleep 1']
[2025-02-28T04:48:35.407+0000] {subprocess.py:99} INFO - Output:
[2025-02-28T04:48:35.409+0000] {subprocess.py:106} INFO - example_bash_operator__runme_2__20250228
[2025-02-28T04:48:36.413+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-28T04:48:36.452+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-28T04:48:36.454+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=runme_2, run_id=manual__2025-02-28T04:48:26+00:00, execution_date=20250228T044826, start_date=20250228T044835, end_date=20250228T044836
[2025-02-28T04:48:36.492+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-02-28T04:48:36.493+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-02-28T04:48:36.494+0000] {logging_mixin.py:190} INFO - Dag name:example_bash_operator queued_at:2025-02-28 04:48:32.304790+00:00
[2025-02-28T04:48:36.494+0000] {logging_mixin.py:190} INFO - Task hostname:151d4861bf1b operator:BashOperator
[2025-02-28T04:48:36.526+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-28T04:48:36.537+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
